src/controllers/controller.bar.js

```diff
@@ -412,6 +412,11 @@ export default class BarController extends DatasetController {
     let length = _stacked ? me.applyStack(vScale, parsed, _stacked) : value;
     let head, size;
 
+    if (isNullOrUndef(length)) {
```

Comment:
Wondering if the issue should be resolved at a deeper level, does it happen with line charts too?

Reply:
My initial fix was to change https://github.com/chartjs/Chart.js/blob/master/src/core/core.datasetController.js#L75 to `return 0` which worked but broke a stacked line filler test. Let me see if line charts have the same problem

Reply:
What if it returned the `value`?

Reply:
https://github.com/chartjs/Chart.js/blob/7fab66ccd1637808868fc8b70b753642e82bd3b3/src/core/core.datasetController.js#L74-L76

Thinking that could be:

```js
  if (isNullOrUndef(value)) {
    return value;
  }
```

Reply:
Line charts exhibit the same problem, so will need a fix as well. Investigating that now.

Reply:
Ok, so that proposed changed works for bars, but doesn't work for lines. I'll update my fix and look further at the line chart

Reply:
oh, ok. feel free to ignore the suggestion, as it did not work for both.

Reply:
ok 👍 will keep bars as is, because the `return value` change would fail if the value was ever undefined because of how the type casting works. `null` gets cast to `0` whereas `undefined` doesn't